File: backend/src/main/java/com/campus/Campus_Connect/features/event/dto/response/ParticipantsResponse.java
package com.campus.Campus_Connect.features.event.dto.response;

import lombok.*;

import java.util.List;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class ParticipantsResponse {

    private List<ParticipantTeamResponse> teams;

    private List<SoloParticipantResponse> soloParticipants;

    private Integer totalParticipants;

}

File: android-app/app/src/main/java/com/example/campusconnect/feature/events/data/remote/response/ParticipantsResponse.kt
package com.example.campusconnect.feature.events.data.remote.response

import com.google.gson.annotations.SerializedName

data class ParticipantsResponse(

    @SerializedName("teams")
    val teams: List<ParticipantTeamResponse>,

    @SerializedName("soloParticipants")
    val soloParticipants: List<SoloParticipantResponse>,

    @SerializedName("totalParticipants")
    val totalParticipants: Int
)